README

```diff
@@ -59,16 +59,13 @@ Bugs and contributing
 ----------------------------
 
 Please send bug reports to mailing list:
-	linux-nfc@lists.01.org
+  [linux-nfc@lists.01.org](mailto:linux-nfc@lists.01.org)
```

Comment:
That's a no go. It's README, not markdown.

Reply:
Sorry, not clear on this requested change. Is this with respect specifically to line 62, or the entire PR, since the PR was essentially "README for Github" edits?